napari/_qt/_tests/test_qt_notifications.py

```diff
@@ -88,7 +141,9 @@ def test_notification_manager_via_gui(
     warnButton.clicked.connect(warn_func)
     qtbot.addWidget(errButton)
     qtbot.addWidget(warnButton)
-
+    monkeypatch.setattr(
+        NapariQtNotification, "show_notification", lambda x: None
+    )
```

Comment:
For me, it is only a workaround, but I do not have any idea how to do this better.